account_fiscal_year_closing/models/account_fiscalyear_closing_abstract.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models
+
+
+class AccountFiscalyearClosingAbstract(models.AbstractModel):
+    _name = "account.fiscalyear.closing.abstract"
+
+    name = fields.Char(string="Description", required=True)
+    company_id = fields.Many2one(
+        comodel_name='res.company', string="Company", ondelete='cascade',
+    )
+    check_draft_moves = fields.Boolean(
+        string="Check draft moves", default=True,
+        help="Checks that there are no draft moves on the fiscal year "
+             "that is being closed. Non-confirmed moves won't be taken in "
+             "account on the closing operations.",
+    )
+
+
+class AccountFiscalyearClosingConfigAbstract(models.AbstractModel):
+    _name = "account.fiscalyear.closing.config.abstract"
+    _order = "sequence asc, id asc"
+
+    name = fields.Char(string="Description", required=True)
+    sequence = fields.Integer(string="Sequence", index=True, default=1)
+    code = fields.Char(string="Unique code", required=True)
+    inverse = fields.Char(
+        string="Inverse config",
+        help="Configuration code to inverse its move",
+    )
+    reconcile = fields.Boolean(
+        string="Reconcile", help="Reconcile inverse move",
+    )
+    move_type = fields.Selection(
+        selection=[
+            ('closing', 'Closing'),
+            ('opening', 'Opening'),
+            ('loss_profit', 'Loss & Profit'),
+            ('other', 'Other'),
+        ], string="Move type", readonly=True, default='closing',
```

Comment:
I suggest to remove the readonly=True.  In case that you want to create a manual closing template you'd want to change this value, and it is now not possible. For instance, now to test I had to load the template from CSV.

Reply:
OK about this.